src/portal/azext_portal/generated/_params.py

```diff
@@ -0,0 +1,62 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from argcomplete.completers import FilesCompleter
+from azure.cli.core.commands.parameters import (
+    tags_type,
+    resource_group_name_type,
+    get_location_type,
+    file_type
+)
+from azure.cli.core.commands.validators import get_default_location_from_resource_group
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('portal dashboard list') as c:
+        c.argument('resource_group_name', resource_group_name_type,
+                   help='The name of the resource group.')
+
+    with self.argument_context('portal dashboard show') as c:
+        c.argument('resource_group_name', resource_group_name_type,
+                   help='The name of the resource group.')
+        c.argument('name', options_list=[
+                   '--name', '-n'], help='The name of the dashboard.')
+
+    with self.argument_context('portal dashboard create') as c:
+        c.argument('resource_group_name', resource_group_name_type,
+                   help='The name of the resource group.')
```

Comment:
Plese remove the help text. `resource_group_name_type` has provided a more detailed help message.